src/js/notifications.js

```diff
@@ -0,0 +1,67 @@
+var Wand = (function(wand) {
+
+/* Inspiration taken from
+   https://github.com/cugos/dropchop/blob/master/src/js/notifications/Notifications.js
+ */
+
+  'use strict';
+
+  wand = wand || {};
+  wand.notifications = {};
+  var params;
+
+  wand.notifications.init = function() {
```

Comment:
I'm not sure. Order is important in terms of initialization of things, since right now everything is tied to `wand.elem`. Perhaps that doesn't make sense.